### PR TITLE
EUI-5729 initial commit

### PR DESCRIPTION
**Pull request by tarikdogan-hmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-5729

### Change description ###

Specific access request node layer part 2

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: api/specificAccessOrchastrator/index.ts [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+async function specificAccessRequestCreateAmRoleRoleback(req, res, next): Promise<Response> {
+  try {
+    return res.status(201).send({roleAssignmentId:101, rolebackStatus:true});
+  } catch (error) {
+    next(error);
+  }
+}
+
+async function specificAccessRequestCreateTask(req, res, next): Promise<Response> {
```
Comment (udaydenduluri33) [inline api/specificAccessOrchastrator/index.ts:82]: this can be deleted

File: api/routes.ts [hunk 2]
```diff
@@ -43,5 +44,15 @@ router.use('/service-ref-data', serviceRefDataRouter);
 router.use('/prd', prdRouter);
 router.use('/hearings', hearingsRouter);
 
+//http://localhost:3000/am/role-assignments1
```
Comment (udaydenduluri33) [inline api/routes.ts:47]: remove the dead code (comments)

File: api/specificAccessOrchastrator/routes.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import * as express from 'express';
+import authInterceptor from '../lib/middleware/auth';
+// import { specificAccessRequestCreateAmRole } from './index';
+
+export const router = express.Router({mergeParams: true});
+//router.use(authInterceptor);
+
+
+
+//router.post('', specificAccessRequest);
```
Comment (udaydenduluri33) [inline api/specificAccessOrchastrator/routes.ts:11]: please remove the dead code (comments)

File: api/specificAccessOrchastrator/routes.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import * as express from 'express';
+import authInterceptor from '../lib/middleware/auth';
+import { OrchastrationSpecificAccessRequest } from './index';
+
+export const router = express.Router({mergeParams: true});
+
+router.use(authInterceptor);
+router.post('/', OrchastrationSpecificAccessRequest);
```
Comment (udaydenduluri33) [inline api/specificAccessOrchastrator/routes.ts:8]: OrchastrationSpecificAccessRequest Should be camelCase

File: api/routes.ts [hunk 5]
```diff
@@ -42,6 +43,7 @@ router.use('/locations', locationsRouter);
 router.use('/service-ref-data', serviceRefDataRouter);
 router.use('/prd', prdRouter);
 router.use('/hearings', hearingsRouter);
+router.use('/specific-access-request', SpecificAccessRouter);
```
Comment (udaydenduluri33) [inline api/routes.ts:46]: SpecificAccessRouter should be camel case

File: api/proxy.config.ts [hunk 6]
```diff
@@ -128,12 +126,13 @@ export const initProxy = (app: Express) => {
       source: '/api/refund',
       target: getConfigValue(SERVICES_REFUNDS_API_URL),
   });
-  applyProxy(app, {
-    onReq: accessManagement.removeAcceptHeader,
-    rewrite: false,
-    source: '/am/role-assignments',
-    target: getConfigValue(SERVICES_ROLE_ASSIGNMENT_API_PATH),
-  });
+  // applyProxy(app, {
```
Comment (udaydenduluri33) [inline api/proxy.config.ts:129]: remove dead code

File: api/specificAccessOrchastrator/routes.ts (hunk 4 above)
Comment (udaydenduluri33) [reply api/specificAccessOrchastrator/routes.ts:8]: spell check for OrchastrationSpecificAccessRequest

**Review by udaydenduluri33 — APPROVED** [review]
@tarikdogan-hmcts  thanks for making the changes

**Review by waynepHMCTS — APPROVED** [review]

**Pull request merged**